assets/i18n/qqq.json

```diff
@@ -296,7 +296,14 @@
 	"I18N_PLAYER_FEEDBACK_TOOLTIP": "Text displayed when the user is playing an exploration. - Tooltip for the button that opens the a dialog to send feedback to the authors of the exploration.\n{{Identical|Feedback}}",
 	"I18N_PLAYER_FORWARD": "Text read to users with screenreaders when they navigate through an exploration. - This labels the rightward-pointing arrow that is used to advance forward by one card in the exploration.\n{{Identical|Forward}}",
 	"I18N_PLAYER_GO_TO_LIBRARY": "Text displayed when the user is playing an exploration. - Text shown inside a button when the user finishes playing an exploration. When the user clicks the button, the library page is loaded.",
-	"I18N_PLAYER_INFO_TOOLTIP": "Text displayed when the user is playing an exploration. - Tooltip for the button that opens the a dialog with more information about the exploration.\n{{Identical|Info}}",
+    "I18N_PLAYER_HINT": "Text displayed when the user is playing an exploration. - Text shown inside a button when a hint is available for the card being viewed. When the user clicks the button, a hint is displayed in the conversation interface.",
```

Comment:
Oh, I just noticed something. For translation files you might need to use tabs (or do whatever the surrounding lines do). That's what translatewiki expects.